Python/dtoa.c

```diff
@@ -351,7 +346,13 @@ typedef struct Bigint Bigint;
    Bfree to PyMem_Free.  Investigate whether this has any significant
    performance on impact. */
 
-static Bigint *freelist[Kmax+1];
+
+/* Get Bigint freelist from interpreter  */
+static Bigint **
+get_freelist() {
```

Comment:
See "function declaration isn’t a prototype [-Wstrict-prototypes]" compiler warning:
```suggestion
get_freelist(void) {
```

Reply:
Thank you for your review, sorry I didn't pay attention to the warning.